projects/chaos/src/power_path.c

```diff
@@ -132,3 +152,24 @@ StatusCode power_path_read_source(const PowerPathSource *source, PowerPathVCRead
   *values = source->readings;
   return STATUS_CODE_OK;
 }
+
+bool power_path_process_event(PowerPathCfg *cfg, const Event *e) {
+  PowerPathSource *source = NULL;
+  if (e->data == POWER_PATH_SOURCE_ID_DCDC) {
+    PowerPathSource *source = &cfg->dcdc;
```

Comment:
Is there a reason why you are assigning this scoped to the `if` block? You're going to be passing `NULL` to `power_path_source_monitor_enable` and `power_path_source_monitor_disable`

Reply:
Oops. Done.